### PR TITLE
Feat/contributors generation

### PR DESCRIPTION
**Pull request by sighphyre**
This includes the SVG fork for Unleash contributors generation and extends the limit of the contributors to 500. Beyond 500 contributors Github no longer returns user info, so this may be a limit going forward.

### CODE REVIEW

File: .github/workflows/update_contributors.yaml [hunk 1]
```diff
@@ -10,18 +10,24 @@ jobs:
...
     - uses: aws-actions/configure-aws-credentials@v1
       with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
     - name: Publish static assets to S3
       run: |
-        aws s3 cp ./contributors.svg s3://getunleash-static/docs-assets/contributors.svg
\ No newline at end of file
+        ls -l
```
Comment (thomasheartman) [inline .github/workflows/update_contributors.yaml:32]: Why are we listing files? Is this a debugging leftover or does it solve some weird (or not-so-weird) issue?

File: .github/workflows/update_contributors.yaml [hunk 2]
```diff
@@ -10,18 +10,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - name: Use Node.js 14.x
+    - name: Use Node.js 16.x
```
Comment (thomasheartman) [inline .github/workflows/update_contributors.yaml:13]: Like it! Modern 🤖

**Review by thomasheartman — APPROVED** [review]
Looks good to me, though I think you _might_ have left some debugging code in there by mistake? Added a comment inline, anyway 💁🏼

As for the actual creation: once this is in, we may want to start experimenting with how large the avatars are (and of how many per column -- maybe I need to create a PR towards your fork 🤔 ), but we'll do that once we see how things work out.

File: .github/workflows/update_contributors.yaml (hunk 1 above)
Comment (sighphyre) [reply .github/workflows/update_contributors.yaml:32]: Sorry I missed this comment! Yeah this is debugging info, you're at the mercy of the CDN to confirm the correct image was build, which is an hour long time out. This is super useful to check against a local image to confirm the correct image was built

**Pull request merged**